pc_ble_driver_py/ble_driver.py

```diff
@@ -1498,6 +1498,11 @@ def __init__(self, uuid, handle):
     @classmethod
     def from_c(cls, gattc_desc):
         return cls(uuid=BLEUUID.from_c(gattc_desc.uuid), handle=gattc_desc.handle)
+    
+    def __eq__(self, other):
+        if isinstance(other, BLEDescriptor):
+            return self.uuid == other.uuid
+        return self.uuid == other
```

Comment:
This way of using the __eq__ operator looks a bit iffy to me. I would not expect 

`obj1 == obj2`
and 
`obj1.field1 == obj2` 

to both evaluate to `True`

If you must check if an element is in a list, you can use list-comprehensions to do it, even if they tend to be a bit uglier.